src/ImageRunModal.scss

```diff
@@ -0,0 +1,18 @@
+@import '@patternfly/patternfly/layouts/Grid/grid.css';
+
+// Limit the max-width more the override in podman.scss which is 40%
+#run-image-dialog-memory, #run-image-cpu-priority {
+     max-width: 15%;
+}
+
+#memory-unit-select {
+     max-width: 20%;
+}
```

Comment:
Generally, it's a bad idea to use IDs in CSS. You want to use classes for CSS, IDs for JavaScript, and not mix them.

This is due to specificity in CSS and speed in targeting IDs in JavaScript. There are a few exceptions for one-offs, like this, sometimes. But it's usually still better to duplicate the ID's value as the name of the class (for the resulting HTML... in our case, the elements in JSX which eventually become HTML) and use the class in the CSS.

Reply:
Thanks, I applied your suggestions and changed it to classes everywhere.